updater/lib/dependabot/setup.rb

```diff
@@ -10,8 +10,8 @@
 end
 
 require "dependabot/sentry"
-Raven.configure do |config|
-  config.logger = Dependabot.logger
+Sentry.init do |config|
+  # config.logger = Dependabot.logger # TODO
```

Comment:
TODO: still need to research if this should be replaced with anything, or simply deleted...